src/algorithms/fista.h

```diff
@@ -89,7 +89,7 @@ Results Family::fitFISTA(const T& x, const mat& y, vec lambda)
     if (optimal && feasible){
       // Either intercept term is absent or
       // gradient with respect to intercept is low
-      if(!intercept || std::abs(grad(0,0))<1e-2)
+      if(!intercept || abs(grad.row(0)).max() < 1e-2)
```

Comment:
Hm, is this really a reasonable tolerance threshold?

Reply:
This is just a quick fix I wrote way back when intercept term wasn't matching with ADMM. I noticed that SLOPE package now uses a different criteria (which works way better than this) so I figured this wasn't going to be used anyway when we will merge into the SLOPE package.